dataland-backend/src/test/kotlin/org/dataland/datalandbackend/validator/ExtendedNumberDataPointValidationTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.dataland.datalandbackend.validator
+
+import org.dataland.datalandbackend.model.datapoints.ExtendedDataPoint
+import org.dataland.datalandbackend.model.enums.data.QualityOptions
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class ExtendedNumberDataPointValidationTest {
+    val numberDataPointValidator = DataPointMinimumValidator()
+    val dataPointWrongType = ExtendedDataPoint<Double>(12.1, QualityOptions.Reported)
+
+    @Test
+    fun testIfExceptionIsThrownForWrongDatatype() {
+        val errorThrown = assertThrows<Exception> {
```

Comment:
Ok the error is expected because it is an ExtendedDataPoint<Double> which we don't expect, correct?

Reply:
the error is thrown, because the value of extendedDataPoint is of type Double, but only Long and BigDecimal are supported

Reply:
ok